resolution_ratio:
      DCX_26 : 1024
      DCX_32 : 1024
      RE_40  : 500

reduction_ratio:
      motor_type_slave_3:
                          - 100
                          - 100
                          - 360
                          - 250
                          - 200
                          - 200
                          - 250
                          - 200
                          - 200

      motor_type_slave_4:
                          - 360
                          - 360
                          - 360
                          - 360
                          - 100
                          - 160
                          - 360
                          - 360
                          - 384

      motor_type_slave_5:
                          - 1
                          - 1
                          - 100
                          - 384
                          - 360
                          - 1
                          - 160
                          - 100
                          - 100


